app/src/common/shared/org/mozilla/vrbrowser/telemetry/GleanMetricsService.java

```diff
@@ -0,0 +1,63 @@
+package org.mozilla.vrbrowser.telemetry;
+
+import android.content.Context;
+import android.util.Log;
+
+import androidx.annotation.UiThread;
+
+import org.mozilla.vrbrowser.browser.SettingsStore;
+import org.mozilla.vrbrowser.utils.DeviceType;
+import org.mozilla.vrbrowser.utils.SystemUtils;
+import org.mozilla.vrbrowser.BuildConfig;
+import org.mozilla.vrbrowser.GleanMetrics.Metrics;
+
+import mozilla.components.service.glean.Glean;
+import mozilla.components.service.glean.config.Configuration;
+
+
+public class GleanMetricsService {
```

Comment:
it's just in order to manage Glean and its metrics easily. I would expect I will put the metrics we need in this wrapper then.

Reply:
I would be a bit cautious about this: Fenix did this (see their events wrapping) and had/has troubles. This is just adding an extra layer of abstraction, but none is really needed here, since the other collection library you're using will go away.

Anyway, I don't feel too strongly about this in its current form.